src/Tools/dotnet-symbol/Program.cs

```diff
@@ -381,9 +383,18 @@ private IEnumerable<SymbolStoreKeyWrapper> GetKeys()
                     }
                     foreach (SymbolStoreKeyWrapper wrapper in generator.GetKeys(flags).Select((key) => new SymbolStoreKeyWrapper(key, inputFile)))
                     {
+                        count++;
                         yield return wrapper;
                     }
                 }
+                if (count > 0)
+                {
+                    Tracer.Verbose($"Generated {count} SymbolStoreKeys for `{inputFile}`.");
+                }
+                else
+                {
+                    Tracer.Verbose($"Could not generate SymbolStoreKeys for `{inputFile}`.");
```

Comment:
This helps, but I'd still imagine people being confused. They'll know file XYZ didn't work but they probably won't know why. Perhaps some more explicit logic around the PE, ELF, and MachO key generators could explicitly say when the binary doesn't include symbol information?